packages/framework/pr_tools/PullRequestLinuxDriverTest.py

```diff
@@ -109,6 +109,13 @@ def parse_args():
                           help='The Jenkins build number',
                           required=True)
 
+    optional.add_argument('--dashboard-build-name',
+                          dest="dashboard_build_name",
+                          action='store',
+                          default="UNKNOWN",
```

Comment:
Why not default to None?

Reply:
This was set to `UNKNOWN` based on other defaults for optional arguments.

The other reason was that I initially wanted the reading of the Jenkins envvar `DASHBOARD_BUILD_NAME` to also default to `UNKNOWN` when the value is not set with something like [`${DASHBOARD_BUILD_NAME:=UNKNOWN}`](https://github.com/trilinos/Trilinos/blob/9bd04b9afdeab2dacf2ad14abe43576af464a60f/packages/framework/pr_tools/PullRequestLinuxDriver.sh#L213) and have a check for this for the Nightly track to not disrupt the names for Nightly jobs without envvar set.
However, figured that the envvars should be set prior to merging this anyways so that shouldn't be a worry and left this default.